robyn/processpool.py

```diff
@@ -180,8 +180,8 @@ def spawn_process(
     for middleware_type, middleware_function in global_middlewares:
         server.add_global_middleware(middleware_type, middleware_function)
 
-    for route_type, endpoint, function in route_middlewares:
-        server.add_middleware_route(route_type, endpoint, function)
+    for middleware_type, endpoint, route_type, function in route_middlewares:
+        server.add_middleware_route(middleware_type, endpoint, function, route_type)
```

Comment:
the arrangement order is not nice. 

@VishnuSanal , could you make it identical to the lines above.

i.e. something like

```
    for middleware_type, endpoint, function, route_type in route_middlewares:
        server.add_middleware_route(middleware_type, endpoint, function, route_type)
``` 

Like this or something?